### ISSUE TITLE
WixSharp.Core appears to be missing XML docs

### ISSUE THREAD

**Issue by BobVul** [oleg-shilo/wixsharp#1678]
I'm attempting to use WixSharp.Core against .net8.0-windows7.0, since I would like my installer to be easily editable in VSCode without needing full VS.

However, the XMLdoc documentation is not displayed within the editor. This makes the editing experience somewhat difficult.

From a quick look at the nupkg, it appears that the xml files are not included in the Core packages?

Would it be possible to fix this?

**Comment by oleg-shilo** [comment]
Yep, the check box for XML documentation was not checked. Not sure how this happened.

Done. Fixed now. I have republished the packages v2.4.2.1

**Comment by BobVul** [comment]
Thank you, 2.4.2.1 docs work fine.
